src/Zynga/Framework/PgData/V1/PgModel/Reader.hh

```diff
@@ -74,7 +75,8 @@ class Reader implements ReaderInterface {
       $where->and($obj->getPrimaryKey(), PgWhereOperand::EQUALS, $id);
 
       // 3) Run the query against the database and dont release any of the lock yet
-      $resultSet = $this->fetchResultSetFromDatabase($model, $where, false);
+      $resultSet =
+        $this->fetchResultSetFromDatabase($model, $where, false, false);
```

Comment:
this shouldn't be false and should be the $this->_pgModel-><functionName>(). though i am not sure if you should even pass it in here. let the function below directly read it via pgmodel?